modules/ROOT/pages/authentication-authorization/dbms-administration.adoc

```diff
@@ -576,27 +672,36 @@ SHOW ROLE userModifier PRIVILEGES AS COMMANDS
 a|Rows: 1
 |===
 
-A user that is granted the `ALTER USER` privilege is allowed to run the `ALTER USER` administration command with one or several of the `SET PASSWORD`, `SET PASSWORD CHANGE [NOT] REQUIRED`, `SET AUTH`, `REMOVE AUTH` and `SET STATUS` parts:
+
+The `ALTER USER` privilege allows the user to run the `ALTER USER` administration command with one or several of the `SET PASSWORD`, `SET PASSWORD CHANGE [NOT] REQUIRED`, `SET AUTH`, `REMOVE AUTH` and `SET STATUS` parts. +
```

Comment:
not sure why it wasn't there before either but this should probably also mention the set/remove home database clauses